presto-main/src/test/java/com/facebook/presto/type/TestCharOperators.java

```diff
@@ -90,6 +100,11 @@ public void testGreaterThan()
         assertFunction("cast('bar' as char(3)) > cast('foo' as char(3))", BOOLEAN, false);
         assertFunction("cast('foobar' as char(6)) > cast('foobaz' as char(6))", BOOLEAN, false);
         assertFunction("cast('foob r' as char(6)) > cast('foobar' as char(6))", BOOLEAN, false);
+        assertFunction("cast(' ' as char(1)) > cast('\0' as char(1))", BOOLEAN, true);
```

Comment:
\0 is < than ' ' there is no latter < than ' ', that's why this test is different than others thus important